src/app/containers/MediaPlayer/Metadata.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import Helmet from 'react-helmet';
-import { shape } from 'prop-types';
-import mediaPlayerMetadata from './helpers/metadata';
+import pathOr from 'ramda/src/pathOr';
+import { shape, string } from 'prop-types';
+import { mediaPlayerMetadata } from './helpers/metadata';
 
-const Metadata = ({ aresMediaBlock }) => {
-  const metadata = mediaPlayerMetadata(aresMediaBlock);
+const Metadata = ({ aresMediaBlock, embedSource }) => {
+  const aresMediaBlocks = pathOr(null, ['model', 'blocks'], aresMediaBlock);
+
+  if (!aresMediaBlocks || aresMediaBlocks.length < 1) {
+    return null;
```

Comment:
this line isn't been covered, could we also update the test for this file to simulate the condition for the null return

Reply:
Addressed this case scenario in this unit test https://github.com/bbc/simorgh/blob/e1144867365b0978dac6d419219d39ffc270523c/src/app/containers/MediaPlayer/helpers/metadata/index.test.js#L110/